Makefile

```diff
@@ -365,7 +403,7 @@ storybook-build: | storybook-configure
 
 run-storybook: storybook-build
 	echo -e "\033[92mRunning:\033[39m Storybook"
-	$(STORYBOOK_BUILD_PATH)/bin/Storybook
+	nixGL $(STORYBOOK_BUILD_PATH)/bin/Storybook
```

Comment:
Same here, enforcing nix is harsh...

Reply:
`nixGL` is not enforcing. It's a wrapper, which fixed running an app in Nix shell with opengl hardware acceleration.
`check-nix-shell` is actually enforing Nix shell for `nim_status_client` build on Linux.

Reply:
Got it; will sum up my thoughts in a standalone comment